main.go

```diff
@@ -50,6 +50,7 @@ func init() {
 }
 
 var log = logging.Logger("estuary")
+var GitVersion string
```

Comment:
Is the GitVersion different from the App version here? https://github.com/application-research/estuary/pull/242/files

Reply:
Nope